src/Sulu/Component/Websocket/MessageDispatcher/AuthenticatedMessageDispatcherApp.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+namespace Sulu\Component\Websocket\MessageDispatcher;
+
+use Doctrine\Common\Cache\Cache;
+use Ratchet\ConnectionInterface;
+use Sulu\Component\Websocket\ConnectionContext\AuthenticatedConnectionContext;
+use Sulu\Component\Websocket\ConnectionContext\ConnectionContextInterface;
+
+/**
+ * Extends the message dispatcher app with authentication
+ */
+class AuthenticatedMessageDispatcherApp extends MessageDispatcherApp
```

Comment:
Because we cannt share sessions over normal http and websocket when we use nativ filesystem session handler we (me and @chirimoya) decided to inject the id of the user. Therefore we dont need this code at the moment but i dont want to remove it completely because it works with other session handlers ... and i think we will need it in the future! Now we use only the `MessageDispatcherApp`.